src/helpers/idle.ts

```diff
@@ -0,0 +1,62 @@
+import { AbortError, sleep } from '@/helpers/sleep';
```

Comment:
added lazyMount for ActivityList - @christianbaroni maybe not necessary with your lazy tabs work? it's a nice helper to have either way but can remove this if we don't want it.